rogue-thi-app/components/RoomMap.js

```diff
@@ -225,6 +227,17 @@ export default function RoomMap ({ highlight, roomData }) {
             Stattdessen nach Zeitraum suchen
           </a>
         </Link>
+
+        {userKind === USER_STUDENT &&
+          <div>
```

Comment:
Is the `div` required? Otherwise we should just use `<>` (empty element, which will not create a DOM-node)